web/pages/field/multiselect.tsx

```diff
@@ -0,0 +1,176 @@
+import MultiSelect from 'frui/field/MultiSelect';
+import Link from 'next/link';
+import Crumbs, { Crumb } from 'modules/components/Crumbs';
+import { LayoutPanel } from 'modules/theme';
+import { Translate, useLanguage } from 'r22n';
+import Code, { InlineCode as C } from 'modules/components/Code';
+import Props from 'modules/components/Props';
+
+export default function Home() {
+    const { _ } = useLanguage();
+
+    const crumbs: Crumb[] = [
+        { icon: 'rectangle-list', label: 'Fields', href: '/field' },
+        { label: 'Multi Select' }
+    ];
+
+    const props = [
+        [ _('className'), _('string'), _('No'), _('Standard HTML class names') ],
+        [ _('options'), _('string[]'), _('Yes'), _('An array of strings') ],
+        [ _('custom'), _('boolean'), _('No'), _('Allow users to add custom options.') ],
+        [ _('searchable'), _('boolean'), _('No'), _('Allow users to search for options.') ],
+        [ _('name'), _('string'), _('No'), _('Used for react server components.') ],
+        [ _('placeholder'), _('string'), _('No'), _('Placeholder text to be displayed.') ],
+        [ _('style'), _('CSS Object'), _('No'), _('Standard CSS object') ],
+    ];
+
+    return (
+        <LayoutPanel>
+            <main className='flex flex-col h-full w-full'>
+                <div className='p-3 bg-b2'>
+                    <Crumbs crumbs={crumbs}/>
+                </div>
+                <div className='flex-grow relative h-full'>
```

Comment:
React attributes should use double quotes. Invalid classnames. Please do not use tailwind classnames.